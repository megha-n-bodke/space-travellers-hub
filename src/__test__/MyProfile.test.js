import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import '@testing-library/jest-dom';
import store from '../redux/configureStore';
import Profile from '../components/my_profile/MyProfile';

it('Check any changes to the component', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <Profile />
    </Provider>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

it('the component should render a div container', () => {
  render(
    <Provider store={store}>
      <Profile />
    </Provider>,
  );

  const rocket = screen.getByTestId('container');

  expect(rocket).toBeInTheDocument();
});
