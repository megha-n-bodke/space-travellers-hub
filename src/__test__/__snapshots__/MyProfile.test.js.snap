// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check any changes to the component 1`] = `
<div
  className="container"
  data-testid="container"
>
  <div
    className="reserve"
  >
    <tr>
      <th>
        <h2
          className="title"
        >
          My Reservations
        </h2>
      </th>
    </tr>
    <tr />
  </div>
  <div
    className="reserve"
  >
    <tr>
      <th>
        <h2
          className="title"
        >
          My Reservations
        </h2>
      </th>
    </tr>
    <tr />
  </div>
</div>
`;
